### PR TITLE
added optional maker-bundle if PHP >= 7.0

### PR DESCRIPTION
**Pull request by OskarStark**
Should be merged after https://github.com/sonata-project/SonataAdminBundle/pull/5093/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R70 is green

### CODE REVIEW

**Comment by OskarStark** [comment]
@greg0ire I didn't see your review, as i force push in this moment, can you please review again? Thank you!

File: project/.travis/before_install_test.sh.twig [hunk 1]
```diff
@@ -30,5 +30,15 @@ fi
 sed --in-place "s/\"dev-master\":/\"dev-${TRAVIS_COMMIT}\":/" composer.json
 
 {% for package_name,package_versions in versions if package_versions|length > 0 %}
-if [ "${{ package_name|upper }}" != "" ]; then composer require "{{ packages[package_name] }}:${{ package_name|upper }}" --no-update; fi;
+    {% if 'symfony_maker' == package_name %}
+        # symfony/maker-bundle only works with PHP 7 and higher
+        if [ "$TRAVIS_PHP_VERSION" '=>' '7.0' ]; then
+            # but only with Symfony 3.4 and higher
+            if [ "$SYMFONY" != "" && "$SYMFONY" '>=' '3.4' ]; then
```
Comment (greg0ire) [inline project/.travis/before_install_test.sh.twig]: You could combine with the other if

File: project/.travis/before_install_test.sh.twig [hunk 2]
```diff
@@ -29,6 +29,18 @@ fi
 
 sed --in-place "s/\"dev-master\":/\"dev-${TRAVIS_COMMIT}\":/" composer.json
 
+# symfony/maker-bundle only works with PHP 7 and higher
```
Comment (Soullivaneuh) [inline project/.travis/before_install_test.sh.twig]: A `TODO` to remove it when PHP5 will not be supported at all should be added here.

**Comment by OskarStark** [comment]
rebased

**Comment by OskarStark** [comment]
Thank you 👍

**Pull request merged**